src/templates/app/user/statement.jsx

```diff
@@ -26,14 +26,29 @@ const Statement = () => (
                 <AccountStatisticsBox id='total_withdrawals' heading={it.L('Total withdrawals')} />
                 <AccountStatisticsBox id='net_deposits'      heading={it.L('Net deposits')} />
             </div>
-
             <div id='util_row' className='gr-row gr-padding-10 gr-parent invisible container'>
-                <div className='gr-12 gr-12-m'>
-                    <div className='gr-row gr-row-align-right gr-row-align-left-m gr-row-align-middle'>
+                <div className='gr-8 gr-8-m'>
+                    <div className='gr-row gr-row-align-left gr-row-align-left-m gr-row-align-middle'>
```

Comment:
QS: do we need both `gr-row-align-left` and `gr-row-align-middle`? Don't they add opposite styles?